_drafts/2020-01-01-nyu-cs-transfer-perspective.md

```diff
@@ -16,13 +16,13 @@ programming skills and the recruiting scene is decent. I agree. But here are a f
 
 # Waitlisting
 
-This is a problem with NYU CS as seen by Data Management and Applied Internet Tech with 120 on the waitlist by the end of the registration process for Spring 2021. However, it is also not a foreign [problem](https://dbknews.com/2017/12/11/umd-computer-science-petition-waitlists-iribe-differential-tuition-class-size/) to UMD. I suspect this also happens at other institutions as there is an increase in number of students majoring in CS. So it's unavoidable, especially with the more popular courses or professors. I recall being on the waitlist for introductory courses at UMD until the last day of add/drop period before getting a spot. Solving waitlisting is difficult as the demand for CS courses keeps going up but universities don't quite have the resources to hire more qualified faculty. One possible solution specific to NYU CS was outlined [here](https://blog.torchnyu.com/2020/12/08/waitlisted-professor-allocation.html) but I'm not sure if it can be done anytime soon.
+This is a problem with NYU CS as seen by Data Management and Applied Internet Tech with 120 on the waitlist by the end of the registration process for Spring 2021. However, it is also not a foreign [problem](https://dbknews.com/2017/12/11/umd-computer-science-petition-waitlists-iribe-differential-tuition-class-size/) to UMD or at [Columbia](https://www.facebook.com/columbiaconfessionz/posts/427826941899592). I suspect this also happens at other institutions as there is an increase in number of students majoring in CS. So it's unavoidable, especially with the more popular courses or professors. I recall being on the waitlist for introductory courses at UMD until the last day of add/drop period before getting a spot. The increase in students majoring in CS certainly led to [Limited Enrollment Program](https://www.lep.umd.edu/computerscience.html#current) after my freshman year, which requires student to re-apply to the major after completing the required courses. This is also done at University of Washington as [pre-major](https://www.washington.edu/uaa/advising/degree-overview/majors/). I'm supposed that it is a reasonable solution to keep the program competitive and have enough resources to accommodate current students. I believe the admission into the major is not guaranteed even if you do well in the required courses since they look at your application hollistically. Personally, it seems stressful to me to experience the anxiety similar to that of college application again after only 2 years. Solving waitlisting is difficult as the demand for CS courses keeps going up but universities don't quite have the resources to hire more qualified faculty. One possible solution specific to NYU CS was outlined [here](https://blog.torchnyu.com/2020/12/08/waitlisted-professor-allocation.html) but I'm not sure if it can be done anytime soon.
```

Comment:
> This is a problem with NYU CS as seen by Data Management and Applied Internet Tech with 120 on the waitlist by the end of the registration process for Spring 2021.

This is a little awkward and kinda passive. What if we make the course the subject? We can also split it into two sentences to make it easier to read.

> Two courses, Data Management and Applied Internet Technologies, demonstrate this problem with NYU CS. For the Spring 2021 term, their waitlists stretched to 120 students.

Reply:
> However, it is also not a foreign [problem](https://dbknews.com/2017/12/11/umd-computer-science-petition-waitlists-iribe-differential-tuition-class-size/) to UMD or at [Columbia](https://www.facebook.com/columbiaconfessionz/posts/427826941899592).

Omit the also. I'm pretty sure the "or" should be a "nor" too

Reply:
anytime -> any time